src/index.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-unused-vars */
+import _ from 'lodash';
+import './style.css';
+import updateStatus from './box_status.js';
+import {
+  addNewTask, editTask, deleteTask, clearAllCompleted,
+} from './add_remove.js';
+import { dragStart, dragOver, drop } from './drag.js';
+
+const taskList = document.getElementById('js-todo-list');
+
+let todoList = [];
+
+function render(tasks) {
+  taskList.innerHTML = '';
+
+  tasks.forEach((task) => {
+    const listItem = document.createElement('li');
+    listItem.draggable = true;
+    listItem.classList.add('border', 'flex', 'height', 'pad-left', 'todo-item');
+    listItem.id = task.index;
+
+    const spanItem = document.createElement('span');
+    spanItem.classList.add('flex', 'text-align');
+    const labelItem = document.createElement('label');
+    const textElement = document.createElement('p');
+    textElement.classList.add('pad-left');
+    textElement.innerText = task.description;
+    const menu = document.createElement('i');
+    menu.classList.add('fa', 'fa-ellipsis-v', 'right');
+    menu.addEventListener('click', (e) => {
+      editTask(e, listItem, textElement, spanItem, todoList, task, render);
+      menu.addEventListener('click', () => {
+        deleteTask(task);
+        todoList = JSON.parse(localStorage.getItem('tasks'));
+        const sortedTasks = todoList.sort((a, b) => a.index - b.index);
```

Comment:
- [ ]  I can see the array sort method is repeating many times It may sound good if you put the sort method of one function and be calling the function with a giving parameter.